package.json

```diff
@@ -36,15 +36,15 @@
     "url": "https://github.com/webarkit/ARnft/issues"
   },
   "devDependencies": {
-    "@types/node": "^14.14.10",
+    "@types/node": "^14.14.31",
```

Comment:
i upgraded the @types/node to the 14 LTS version, and downgraded to the typescript version 3.4 as tagged here: https://www.npmjs.com/package/@types/node